manifests/plugin/tasks.pp

```diff
@@ -6,9 +6,13 @@
 # @param cron_line
 #   Cron line defining when the cleanup cron job should run
 #
+# @param backup
+#   Enable creating a backup of cleaned up tasks in CSV format
```

Comment:
Perhaps it should be noted that this is only relevant if `automatic_cleanup` is true? It's sort of there, but being explicit doesn't hurt.